app/models/katello/ping.rb

```diff
@@ -13,6 +13,10 @@ class << self
       def ping(services: SERVICES, capsule_id: nil)
         result = {}
         services.each { |service| result[service] = {} }
+        if capsule_id && SmartProxy.find(capsule_id).pulp3_enabled?
```

Comment:
could capsule_id be nil here?  (Should you default to master if it is?

Reply:
Yes. It can be null. ```curl -k http://localhost:3000/katello/api/ping``` had nil for capsule